packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -575,7 +579,7 @@ class BulletSpacer extends StatelessWidget {
   Widget build(BuildContext context) {
     final ThemeData theme = Theme.of(context);
 
-    TextStyle textStyle;
+    TextStyle? textStyle;
```

Comment:
this can be late instead of nullable

Reply:
nope, because theme.textTheme.bodyText2 can be null